### ISSUE TITLE
bug: Why does my Angular/Ionic form UI go blank during a page scroll?

### ISSUE THREAD

**Issue by venkata685** [ionic-team/ionic-framework#24462]
### Prerequisites

- [X] I have read the [Contributing Guidelines](https://github.com/ionic-team/ionic-framework/blob/main/.github/CONTRIBUTING.md#creating-an-issue).
- [X] I agree to follow the [Code of Conduct](https://ionicframework.com/code-of-conduct).
- [X] I have searched for [existing issues](https://github.com/ionic-team/ionic-framework/issues) that already report this problem, without success.

### Ionic Framework Version

- [X] v4.x
- [ ] v5.x
- [ ] v6.x

### Current Behavior

Angular 10 Form (also using Ionic for elements). I have a select dropdown with a value that is displayed or an input field with the entered value. When the user scrolls the form both (IOS and Android devices) the selected value goes blank until the page scroll is finished. To get this behavior the finger-press must start in the select area. In other words, if the user’s finger starts somewhere outside the select element this value does not disappear. But if the finger-press starts within the select element and then drags for a page scroll the select value goes blank until the page scroll completes.

### Expected Behavior

The form elements like dropdowns, inputs, etc. should not be disappeared during the page scroll.

### Steps to Reproduce

Create a form with some input fields like username, the password for both IOS and Android devices. 
Enter some value in the username field. 
Start scrolling on the page. we see entered value disappear. 
To get this behavior the finger-press must start in the select area. 
In other words, if the user’s finger starts somewhere outside the select element this value does not disappear.
But if the finger-press starts within the select element and then drags for a page scroll the select value goes blank until the page scroll completes.

**Comment by sean-perkins** [comment]
Can you reproduce the issue with the latest version of Ionic (v6) and record a video to help us visualize the issue? Thanks!

**Comment by liamdebeasi** [comment]
Thanks for the issue! This issue is being closed due to the lack of a code reproduction. If this is still an issue with the latest version of Ionic, please create a new issue and ensure the template is fully filled out.

Thank you for using Ionic!
